pkgs/by-name/ra/railway-travel/package.nix

```diff
@@ -0,0 +1,76 @@
+{
+  lib,
```

Comment:
I also don't like this style but that's besides the point.

Reply:
Thanks for pointing it out. I didn't even notice that if I'm honest, I will pay more attention to it next time.

Reply:
This is the style in RFC 166, so not your fault.